codeclimate/systems.yaml

```diff
@@ -82,3 +82,24 @@ systems:
               description: Github organization name
             - name: repo
               description: Github repository name
+
+      add_test_reporter_id:
+        target:
+          system: codeclimate
+          function: api
+        parameters:
+          URL: '{{ .sysData.url }}/project/gh/{{ .ctx.org }}/{{ .ctx.repo }}/envvar'
+          method: POST
+          content: '{"name":"CC_TEST_REPORTER_ID","value":{{ .ctx.test_reporter_id }}"}'
```

Comment:
This will work. If you set the content type header to `application/json`, honeydipper can also accept a data structure like below and encode it to json for you.
```
  content:
    name: CC_TEST_REPORTER_ID
    value: $ctx.test_reporter_id
```